common/core/src/iosMain/kotlin/com/trafi/locations/LocationsApi.kt

```diff
@@ -0,0 +1,28 @@
+@file:Suppress("unused", "EXTENSION_SHADOWED_BY_MEMBER")
```

Comment:
What does this exactly ? 🤗

Reply:
See [previous answer](https://github.com/trafi/maas-components/pull/101#discussion_r565947786)